### DISCUSSION TITLE
how process Mandarin ASR datasets?

### DISCUSSION THREAD

**Discussion by jaffe-fly** [NVIDIA-NeMo/NeMo#2913] [Q&A] [answered]
https://github.com/NVIDIA/NeMo/blob/main/examples/asr/conf/quartznet/quartznet_15x5_zh.yaml

why yaml labels has no number （1,2,3....）

when number in the datasets,do i need remove?

**Comment by titu1994** [comment, marked as answer]
Numbers should be normalized to text, so that the acoustic model is able to spell it out. Inverse text normalization should be used on the output of the ASR model if original numbers are required (it's not a 100% match but it's usually still good). 

For reference, raw numbers generally will not transcribe very well and will have very difficult to diagnose errors in the output transcript

**Reply by jaffe-fly** [reply]
> Numbers should be normalized to text, so that the acoustic model is able to spell it out. Inverse text normalization should be used on the output of the ASR model if original numbers are required (it's not a 100% match but it's usually still good).
> 
> For reference, raw numbers generally will not transcribe very well and will have very difficult to diagnose errors in the output transcript
thank you !!! 
which tutorials can i see Numbers normalized to text and Inverse text normalization should be used on the output of the ASR model ? 
I dont know how to do this.

**Comment by jaffe-fly** [comment]
when dump to json file do i need set like this ```json.dump(metadata, fout, ensure_ascii=False)```,```ensure_ascii=False``` in the func,?

**Reply by VahidooX** [reply]
I don't think you need to set ensure_ascii=False.

**Reply by jaffe-fly** [reply]
ok thank you
